src/Message/OpenMP.h

```diff
@@ -21,12 +21,12 @@
 #if defined(ENABLE_OPENMP)
 #include <omp.h>
 #else
+#include "formic/utils/openmp.h"
+
```

Comment:
This file belongs in concurrency and looks to add only open mp functions none of them should be defined if QMC_OMP=0 since none of them should be called.